matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/Matrix.kt

```diff
@@ -21,6 +21,7 @@ import androidx.lifecycle.ProcessLifecycleOwner
 import androidx.work.Configuration
 import androidx.work.WorkManager
 import com.zhuinden.monarchy.Monarchy
+import mobile.Mobile
```

Comment:
package name is a bit meaningless, is it possible to change it on your library?

Reply:
I'll see what I can do. I probably can.